tests/opensource-suites/ql/visualizations/column/column.test.ts

```diff
@@ -0,0 +1,85 @@
+import {expect} from '@playwright/test';
+
+import QLPage from '../../../../page-objects/ql/QLPage';
+import {openTestPage} from '../../../../utils';
+import datalensTest from '../../../../utils/playwright/globalTestDefinition';
+import {AxisMode} from '../../../../../src/shared';
+import {PlaceholderName} from '../../../../page-objects/wizard/SectionVisualization';
+import {
+    PlaceholderId,
+    RadioButtons,
+    RadioButtonsValues,
+} from '../../../../page-objects/wizard/PlaceholderDialog';
+
+datalensTest.describe('QL', () => {
+    datalensTest.describe('Column chart', () => {
+        datalensTest.beforeEach(async ({page, config}) => {
+            await openTestPage(page, config.ql.urls.NewQLChartWithConnection);
+
+            const qlPage = new QLPage({page});
+
+            await qlPage.setScript(config.ql.queries.dateAndSales);
+            await qlPage.runScript();
+        });
+
+        datalensTest(
+            'Check that fields order does not affect axis settings',
```

Comment:
Please add a tag `@screenshot` to test name. This helps to sort the failed tests and update the screenshots if necessary.